mindsdb_native/libs/phases/model_interface/lightwood_backend.py

```diff
@@ -75,18 +75,15 @@ def _ts_reshape(self, original_df):
         # Make type `object` so that dataframe cells can be python lists
         for i in range(len(df_arr)):
             for hist_col in ob_arr + self.transaction.lmd['tss']['historical_columns']:
-                df_arr[i][hist_col] = df_arr[i][hist_col].astype(object)
+                df_arr[i].loc[:, hist_col] = df_arr[i][hist_col].astype(object)
 
         # Make all order column cells lists
         for i in range(len(df_arr)):
             for order_col in ob_arr + self.transaction.lmd['tss']['historical_columns']:
                 for ii in range(len(df_arr[i])):
-                    try:
-                        df_arr[i][order_col].iloc[ii] = [df_arr[i][order_col].iloc[ii]]
-                    except Exception:
-                        # Needed because of a pandas bug that causes above to fail for small dataframes
-                        label = df_arr[i].index.values[ii]
-                        df_arr[i].at[label, order_col] = [df_arr[i].at[label, order_col]]
+                    # Needed because of a pandas bug that causes above to fail for small dataframes
+                    label = df_arr[i].index.values[ii]
+                    df_arr[i].at[label, order_col] = [df_arr[i].at[label, order_col]]
```

Comment:
I'm awfully afraid the try/except was there for performance reasons, but for now let's change it since we'll have to revamp this bit anyway as we are moving towards optimizing the ts reshape even further.